mods/d2k/rules/defaults.yaml

```diff
@@ -93,7 +93,7 @@
 	MustBeDestroyed:
 	Voiced:
 		VoiceSet: VehicleVoice
-	Carryable:
+	AutoCarryable:
```

Comment:
On second thought an upgrade rule isn't helpful.  This trait was only available to d2k in the last release, and both uses of the traits there are exceptions to what the automated rule could do.